remoteit/desktop
# [DESK-533] Updates to OOB feature

### Description

Updates to OOB feature
check platform for showing feature
check ssid for showing enabled
check for updates more regularly
added segment tracking

### Checklist


### Test plan

1. Use a Raspberry pi image. 
2. Setup a device
3. check that OOB inactive icon is present
4. connect pi to a network with ssid: remote.itPi
5. verify OOB active icon is present after 1 minute or after leaving and coming back to the page
6. verify segment values are present on segment.io

### Ticket

https://remoteit.atlassian.net/browse/DESK-533
